.werft/jobs/build/validate-changes.ts

```diff
@@ -5,8 +5,7 @@ import { JobConfig } from "./job-config";
 export async function validateChanges(werft: Werft, config: JobConfig) {
     werft.phase("validate-changes", "validating changes");
     try {
-        branchNameCheck(werft, config);
-        preCommitCheck(werft);
+        await Promise.all([branchNameCheck(werft, config), preCommitCheck(werft), typecheckWerftJobs(werft)]);
```

Comment:
That's really nice, I bet there are more places that we could do the same thing 🙂